### PR TITLE
Include test case for Ward Number

### PR DESCRIPTION
**Pull request by nehcuy**
Also included update for PPP

### CODE REVIEW

File: docs/team/nehcuy.md [hunk 1]
```diff
@@ -3,26 +3,82 @@ layout: page
 title: Yuchen's Project Portfolio Page
 ---
 
-### Project: checkUp
+## Project: checkUp
 
-"checkUp" is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+"checkUp" is a desktop address book application used for teaching Software Engineering principles.
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java,
+and has about 15 kLoC.
```
Comment (kxrt) [inline docs/team/nehcuy.md:10]: In a future update, you might want to reword this from the stock lines given in the skeleton draft.

File: docs/team/nehcuy.md [hunk 2]
```diff
@@ -3,26 +3,82 @@ layout: page
...
 
 Given below are my contributions to the project.
 
-* **New Feature**: Allows retrieval of patients by floor number and ward number [coming soon]
+### Code contributed
 
-* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=nehcuy&breakdown=true)
+RepoSense: [link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=nehcuy&breakdown=true)  
+Personal merged PRs: [link](https://github.com/AY2223S1-CS2103T-W16-3/tp/pulls?q=is%3Apr+author%3A%40me+is%3Aclosed)
```
Comment (kxrt) [inline docs/team/nehcuy.md:17]: This is a good idea!

File: src/test/java/seedu/address/logic/commands/getcommands/GetMedicationCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+    private Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void equals() {
+        MedicationContainsKeywordsPredicate firstPredicate = new MedicationContainsKeywordsPredicate(
+                Collections.singletonList("paracetamol"));
+        MedicationContainsKeywordsPredicate secondPredicate = new MedicationContainsKeywordsPredicate(
+                Collections.singletonList("ibuprofen"));
```
Comment (kxrt) [inline src/test/java/seedu/address/logic/commands/getcommands/GetMedicationCommandTest.java:36]: might want to pull in these medication names as the static variables defined in PastAppointmentBuilder for better abstraction.

File: src/test/java/seedu/address/logic/parser/getparsers/GetWardNumberCommandParserTest.java [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        userInput = "D329 B567 A01";
+        expectedGetWardNumberCommand = prepareCommand(userInput);
+        assertParseSuccess(parser, userInput, expectedGetWardNumberCommand);
+    }
+
+    @Test
+    public void parse_validArgs_returnsGetWardNumberCommand() {
+        // no leading and trailing whitespaces
+        GetWardNumberCommand expectedGetWardNumberCommand = prepareCommand("D312 F419");
+        assertParseSuccess(parser, "D312 F419", expectedGetWardNumberCommand);
```
Comment (kxrt) [inline src/test/java/seedu/address/logic/parser/getparsers/GetWardNumberCommandParserTest.java:56]: Different ward numbers are used in `GetWarNumberCommandTest` and here. Maybe use a static variable for cohesiveness?

**Review by kxrt — CHANGES_REQUESTED** [review]
Some small changes required for better OOP, but otherwise looks good to go me. Thanks for the comprehensive testing!

File: src/test/java/seedu/address/logic/commands/getcommands/GetMedicationCommandTest.java (hunk 3 above)
Comment (nehcuy) [reply src/test/java/seedu/address/logic/commands/getcommands/GetMedicationCommandTest.java:36]: I think we can rework this issue of pulling the right constants from a new `TestUtilMessages` class. I will provide a temporary fix for this for now.

File: src/test/java/seedu/address/logic/parser/getparsers/GetWardNumberCommandParserTest.java (hunk 4 above)
Comment (nehcuy) [reply src/test/java/seedu/address/logic/parser/getparsers/GetWardNumberCommandParserTest.java]: Changed Ward Numbers in `GetWardNumberCommandParserTest` to ensure standardised use in `GetWardNumberCommandTest`.

File: docs/team/nehcuy.md (hunk 1 above)
Comment (nehcuy) [reply docs/team/nehcuy.md]: Fixed. Thanks for the catch!

**Review by kxrt — APPROVED** [review]
Thanks for incorporating the feedback!

**Pull request merged**